docs/_static/policy/v1.yaml

```diff
@@ -0,0 +1,133 @@
+openapi: 3.1.0
+info:
+  title: Policy backend API specification
+  version: 0.1.0
+servers:
+  - url: "http://localhost:1323"
+paths:
+  /{did}/presentation_definition:
+    parameters:
+      - name: did
+        in: path
+        description: URLEncoded DID.
+        required: true
+        example: did:web:example.com:1
+        schema:
+          type: string
+      - name: scope
+        in: query
+        description: |
+          This is the scope used in the OpenID4VP authorization request.
+          It is a space separated list of scopes.
+        required: true
+        schema:
+          type: string
+    get:
+      summary: Returns a presentation definition for the given DID and scope.
+      description: |
+        The DID is used for tenant selection. Not all tenants will probably support the same scopes.
+        The scope is used as selection criteria for the presentation definition.
+        It could be the case that the presentation definition is not found.
+        In that case the response will be 201 with an empty body.
+      operationId: "presentationDefinition"
+      tags:
+        - policy
+      responses:
+        "200":
+          description: |
+            DID has been found and the scope is supported.
+            If the scope is supported but no presentation definition is required, the response will be 200 with a presentation definition without any input descriptors.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PresentationDefinition'
+        "201":
+          description: The DID is known but the presented scope is not supported.
+        "404":
+          description: DID is not known to the policy backend.
+  /{did}/authorize:
+    parameters:
+      - name: did
+        in: path
+        description: URLEncoded DID. The DID is used for tenant selection.
+        required: true
+        example: did:web:example.com:1
+        schema:
+          type: string
+    post:
+      summary: Authorize a resource request.
+      description: |
+        When an access token is used to request a resource, the resource server needs to know if the access token grants access to the requested resource.
+        The resource server will send a request to the policy backend to check if the access token grants access to the requested resource.
+
```

Comment:
maybe add that the policy backend does not need to check the presentation submission itself? Since that's fairly complex, it should be done by the Nuts node.